spring-cloud-gcp-pubsub/src/test/resources/logback.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+		<layout class="ch.qos.logback.classic.PatternLayout">
+			<Pattern>
+				%d{yyyy-MM-dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
+			</Pattern>
+		</layout>
+	</appender>
+
+	<logger name="io.netty.handler" level="debug"/>
+
+	<logger name="org.springframework.cloud" level="info"/>
+
+	<root level="info">
+		<appender-ref ref="STDOUT" />
+	</root>
+
+</configuration>
```

Comment:
I think we need to add ` <module name="NewlineAtEndOfFile"/>` to our `checkstyle.xml` config: http://checkstyle.sourceforge.net/config_misc.html#NewlineAtEndOfFile

Reply:
created separate issue for this: https://github.com/spring-cloud/spring-cloud-gcp/issues/338